.github/CODEOWNERS

```diff
@@ -1,34 +0,0 @@
-# Learn how to add code owners here:
-# https://help.github.com/en/articles/about-code-owners
-
-*                                             @timneutkens @ijjk @shuding @huozhi @feedthejim
```

Comment:
Should we make keep these as the global owners instead of the `@vercel/next-js` team? 
This would mean having to make the Next.js team code owners of the subdirectories below.

Reply:
It was configured this way because I don't want to be notified for every PR to this repository

Reply:
You can add `@vercel/next-js:optional` at the top level to give everyone permissions to approve, but it sounds like you want to list out the individuals (or create another smaller team) if you want to avoid sending reviews to certain people on the team right now.

Reply:
Yes, Tim mentioned there's some people we don't want to notify with every PR. I made some changes and moved as much as possible to Vercel Spaces.